packages/backend/src/apps/tiles/actions/create-row/index.ts

```diff
@@ -78,7 +78,17 @@ const action: IRawAction = {
       tableId: string
       rowData: { columnId: string; cellValue: string }[]
     }
-    await validateTileAccess($.flow?.userId, tableId as string)
+
+    try {
```

Comment:
reminder: need to only give access to flow owner only (editors cannot use the tile in their pipes)